pythonAPIClient/client.py

```diff
@@ -3,25 +3,30 @@
 import requests
 
 class Client(object):
+
     """
     Represent a CAM2 client application.
     """
     # TODO: corresponding to the auth route
-    @staticmethod
-    def request_token(id, secret):
-        return 'dummy'
+    def request_token(self):
+        url = 'https://cam2-api.herokuapp.com/auth/?'
+        url += 'clientID='+self.id+'&clientSecret='+self.secret
+        response = requests.get(url)
+        if(response.status_code == 200):
+            self.token = response.json()['token']
+            return "OK"
+        else:
+            return str(response.status_code) + "-" + response.json()['message']
```

Comment:
I think we should return some sort of status code instead of the string "OK" if it works correctly.

Also, if the status code isn't 200 maybe we should raise an exception instead of returning a string with the error message.

Reply:
The get token is called by us " the wrapper class" so i thought giving a message would be better